examples/braket_features/Noise_models/Noise_models_on_Rigetti_AspenM1.ipynb

```diff
@@ -0,0 +1,1057 @@
+{
```

Comment:
<p>The link for "Running quantum circuits on QPU devices" is borken. It should be https://github.com/aws/amazon-braket-examples/tree/main/examples/getting_started/2_Running_quantum_circuits_on_QPU_devices</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>